### PR TITLE
Lvnodcdxi 75

### PR DESCRIPTION
**Pull request by skiffua**
Tested equalizer component.

### CODE REVIEW

File: src/client/components/equalizer/equalizer.test.js [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+      const spyPause = jest.spyOn(instance, 'pauseSoundFromFile');
+
+      wrapper.instance().forceUpdate();
+      wrapper.find('Button[value="Pause"]').simulate('click');
+
+      expect(spyPause).toHaveBeenCalled();
+    });
+
+    it('should call function after click on `startStreamButton` button', () => {
+      const spyPause = jest.spyOn(instance, 'startMuteStream');
```
Comment (tonypntsrn) [inline src/client/components/equalizer/equalizer.test.js]: Here should be spyPause?

File: src/client/components/equalizer/equalizer.test.js [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+    });
+
+    it('will test own method `setCanvasToState` which should set in state reference on canvas element', () => {
+      const paramForTest = document.createElement('canvas');
+      const testedFunction = wrapper.instance().setCanvasToState;
+
+      paramForTest.getContext = jest.fn(() => 'fakeCanvas');
+
+      testedFunction(paramForTest);
+      expect(wrapper.instance().state.ctx).toEqual('fakeCanvas');
```
Comment (tonypntsrn) [inline src/client/components/equalizer/equalizer.test.js]: You already have declared wrapper.instance() in upper scope, why don’t you use it here?

Comment (skiffua) [reply src/client/components/equalizer/equalizer.test.js]: used in all test - missed there. Done

File: src/client/components/equalizer/equalizer.test.js (hunk 1 above)
Comment (skiffua) [reply src/client/components/equalizer/equalizer.test.js]: rewrited

**Review by tonypntsrn — APPROVED** [review]

**Review by NadiiaChorna — APPROVED** [review]

**Pull request merged**